pkg/flags/flags.go

```diff
@@ -7,14 +7,16 @@ import (
 
 	"github.com/linkerd/linkerd2/pkg/version"
 	log "github.com/sirupsen/logrus"
+	"k8s.io/klog"
 )
 
-// ConfigureAndParse adds flags that are common to all go processes, and
-// overrides the default flag for glog logging, which we can't disable. This
+// ConfigureAndParse adds flags that are common to all go processes. This
 // func calls flag.Parse(), so it should be called after all other flags have
 // been configured.
 func ConfigureAndParse() {
-	// override glog's default configuration
+	// override klog's default configuration and log to stderr instead of a log
+	// file.
+	klog.InitFlags(nil)
 	flag.Set("logtostderr", "true")
```

Comment:
is `flag.Set("logtostderr", "true")` still necessary if we're no longer using glog?